examples/teleport-usage/Dockerfile

```
@@ -3,6 +3,7 @@ WORKDIR /app
 COPY go.mod go.sum main.go /app/
 RUN go build -o teleport_usage
 FROM debian:buster-slim
```

```suggestion
FROM debian:stable-slim
```

I know you didn't change this, but I am wondering why we are using `buster-slim`. It's the old stable release of Debian, let's switch to `stable-slim` if we want to stay with a Debian base.